README.md

```diff
@@ -6,35 +6,35 @@ This is the starting point for joining and contributing to the Open Services Gro
 
 ## Meta Information
 
-The way how we organise our community and work is heavily inspired by the Kubernetes Community!
+The way how we organise our community and work is heavily inspired by the [Kubernetes Community]!
 
 ## Governance
 
 Open Services Group has the following types of groups that are officially supported:
 
-* **Steering Committee** are named people that are chartered to take on sensitive topics.
+* **[Steering Committee](/sig-list.md)** are named people that are chartered to take on sensitive topics.
```

Comment:
should this be a different file than the sig-list? A `steering-list.md`

Reply:
yes, should be, needs to be created, thanks